About.html

```diff
@@ -0,0 +1,149 @@
+                </li>
+            </ul>
+            <!-- Ask quote button displayed on the navbar-->
+            <div class="ask-quote">
+                <a href="#">Ask for a quote</a>
+            </div>
+        </nav>
+
+        <!-- Vector background Top and middle -->
+        <div class="chest-bg-vector"></div>
+        <div class="C-bg-vector"></div>
+        <div class="Cross-bg-vector"></div>
+        <div class="0-bg-vector"></div>
+        <div class="cercle-bg-vector"></div>
+
+        <!-- Headline and introduction paragraph -->
+        <section class="headline-container">
+            <h1>Build Safe Let us<br>
+                 Build and Maintain your dream house</h1>
+                <div class="headline-paragraph-container">
+                <p>Our company can intervene in all trades
+                    or in separate trades according to the choices of the client
+                    or the project manager.Lorem Ipsum is simply dummy text of 
+                    the printing and typesetting industry. Lorem Ipsum has been 
+                    the industry's standard dummy.Lorem Ipsum is simply dummy text
+                     of the printing and typesetting industry.</p>
+                </div>
+                    <h3>Build Quality (@2022 Mass)</h3>
+                    
+                    <!-- Vector background bottom -->
+                     <div class="dollar-bg-vector"></div>
+                     <div class="Cross-bg-vector-bottom"></div>
+        </section>
+    </div>
+    </header>
+
+    <!-- Main section containing the ,About Us and the work realized-->
+    <main>
+        <section class="About-us-container">
+        <h4 class="section-title">About Us</h4>
+        <div class="section-title-underline-bar"></div>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+            Lorem Ipsum has been the industry's standard dummy.Lorem Ipsum is simply 
+            dummy text of the printing and typesetting industry. Lorem Ipsum has been 
+            the industry's standard dummy text ever since the 1500s</p>
+        <div class="about-us-logo">Home Maintain .</div>
+        </section>
+    
+    <section class="project-done-container">
+        <h4 class="section-title">Complete Projects</h4>
+        <div class="section-title-underline-bar"></div>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+            Lorem Ipsum has been the industry's standard dummy</p>
+            <div class="project-done">
+        <div class="project-done-1">
+                <p>Project Realized in 2018</p>
+            <div class="color-overlay"></div>    
+            </div>
+        
+            <div class="project-done-2">
+                <p>Project Realized in 2021</p>
+            <div class="color-overlay"></div>    
+            </div>
+        </div>
+    </section>
+
+   <!-- The Partner section -->
```

Comment:
- [ ] You have done an amazing job here, your project is looking really great! Let's just fix one thing and you will be done:

|Desktop|Mobile|
|---|---|
|![Captura de pantalla 2022-05-27 a la(s) 3 20 35 p  m](https://user-images.githubusercontent.com/83717285/170784738-d28b58b8-69e9-413c-8045-202986b506da.png)|![Captura de pantalla 2022-05-27 a la(s) 3 20 48 p  m](https://user-images.githubusercontent.com/83717285/170784788-ef3d9cb1-0afe-4123-836a-60c3c13735b4.png)|

- [ ]  Footer is looking great! However, as you can see in the template, the background changes from light to dark in the mobile version, let's add the dark version.

- [ ] On the About page, the Partners section should be hidden in the desktop version, it would be great to add this feature.

Reply:
Hi @smunozmo 
I have made all the changes suggested by you
thanks for the review